datasets/at/nbter_sanctions/crawler.py

```diff
@@ -0,0 +1,89 @@
+import csv
+import re
+from typing import Dict, List
+import zavod.helpers as h
+from zavod import Context
+
+DATE_FORMATS = ["%d.%m.%Y", "%d. %m %Y"]
+MONTHS_DE = {
+    "Januar": 1,
+    "Februar": 2,
+    "März": 3,
+    "April": 4,
+    "Mai": 5,
+    "Juni": 6,
+    "Juli": 7,
+    "August": 8,
+    "September": 9,
+    "Oktober": 10,
+    "November": 11,
+    "Dezember": 12,
+}
+
+
+def parse_date_time(text: str) -> List[str]:
+    if not text:
+        return None
+    for de, number in MONTHS_DE.items():
+        # Replace German month names with numbers
+        text = re.sub(rf"\b{de}\b", str(number) + ".", text)
+    text = text.replace(" ", "")
+    date = h.parse_date(text, DATE_FORMATS)
+    return date
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    full_name = row.pop("name")
+    # Split `other_name` on `/` and trim any extra whitespace
+    other_names = row.pop("other name").split("/")
+    alias = row.pop("alias")
+    notes = row.pop("notes")
+    reason = row.pop("reason")
+    birth_date = parse_date_time(row.pop("date of birth"))
+    country = row.pop("country")
+    birth_place = row.pop("place of birth")
+    id_number = row.pop("ID card no.")
+    source = row.pop("source")
+    entity_type = row.pop("type")
+
+    entity = None
+    if entity_type == "Person":
+        entity = context.make("Person")
+        entity.id = context.make_id(full_name, birth_place, birth_date)
+        entity.add("name", full_name)  # still need to split it
```

Comment:
what splitting is still needed? no need to split on comma and change the order.
if there's nothing more to do, you can delete the comment.